### PR TITLE
ANC Subsequent visits - Medicate patient that did not get medication at HIV group

### PR DESCRIPTION
**Pull request by anvmn**
#279

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-05-26 20-53

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 1]
```diff
@@ -395,9 +395,10 @@ expectTreatmentReviewTask assembled task =
             True
 
         TreatmentReviewHIV ->
-            -- Show, if there was medication treatment.
-            latestMedicationTreatmentForHIV assembled
-                |> isJust
+            -- Show, if there was medication treatment, or if patient was
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm:398]:
```suggestion
            -- Show, if there was medication treatment, or if the patient was
```

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 2]
```diff
@@ -2911,21 +2928,22 @@ resolveMedicationTreatmentFormInputsAndTasks language currentDate setBoolInputMs
...
-                        |> List.head
-                        |> Maybe.map hivProgramAtHC
-                        |> Maybe.withDefault False
+                referredToHIVProgram =
+                    referredToHIVProgramPreviously assembled
             in
-            if referredToHIVProgram then
+            -- There's a scenarion when there's HIV program at HC,
+            -- patient is referred there, but did not get medication there.
+            -- In this case we'll find this out at Treatment review, and
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm]:
```suggestion
            -- In this case, we'll find this out at the Treatment review, and
```

**Review by nedSaf — APPROVED** [review]
Looks good, minor changes in the comments.

Fix and MWG

**Pull request merged**